conda-forge.yml

```diff
@@ -1,7 +1,6 @@
-provider: {osx: circle, win: azure}
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
-  pkg_format: '2'
```

Comment:
You should not delete this line (it's necessary to produce `.conda` outputs, the newer package format).